tasks/seal/task_porep.go

```diff
@@ -191,6 +191,18 @@ func (p *PoRepTask) GetSpid(db *harmonydb.DB, taskID int64) string {
 	return spid
 }
 
+func (p *PoRepTask) GetSectorID(db *harmonydb.DB, taskID int64) (*abi.SectorID, error) {
```

Comment:
Can we change GetSpid to just return both? This function is a superset of GetSpid

Reply:
After some investigation, I believe we should keep both GetSpid and GetSectorID. GetSpid is widely used, including for certain special tasks that only provide spid, such as window/winning post,.... For other tasks, I can modify GetSpid to obtain spid from GetSectorID. @magik6k What do you think?

Reply:
Yeah, doing this the way you did here looks reasonable